docs/devel/developer-guides/vagrant.md

```diff
@@ -34,45 +34,74 @@ Documentation for other releases can be found at
 
 ## Getting started with Vagrant
 
-Running kubernetes with Vagrant (and VirtualBox) is an easy way to run/test/develop on your local machine (Linux, Mac OS X).
+Running kubernetes with Vagrant (and VirtualBox) is an easy way to
```

Comment:
The last time I tried the vagrant based cluster did not work on OS X. I'd prefer not claiming vagrant as a working option unless we have some CI that tests that it works reliably.